src/Icons/doc/index.rst

```diff
@@ -126,9 +245,154 @@ the ``assets/icons/`` directory. You can think of importing an icon as *locking
 
     Imported icons must be committed to your repository.
 
+Rendering Icons
+---------------
+
+.. code-block:: html+twig
```

Comment:
This section is the one we had before the last doc update. The latest version is this one, which is probably more complete to explain what's really happening when using this function:

```rst
.. code-block:: twig

    {# includes the contents of the 'assets/icons/user-profile.svg' file in the template #}
    {{ ux_icon('user-profile') }}

    {# icons stored in subdirectories must use the 'subdirectory_name:file_name' syntax
       (e.g. this includes 'assets/icons/admin/user-profile.svg') #}
    {{ ux_icon('admin:user-profile') }}

    {# this downloads the 'user-solid.svg' icon from the 'Flowbite' icon set via ux.symfony.com
       and embeds the downloaded SVG contents in the template #}
    {{ ux_icon('flowbite:user-solid') }}
```

Reply:
Thank you, i knew i messed somewhere during the rebase 😅